sklearn/datasets/base.py

```diff
@@ -263,6 +269,9 @@ def load_iris():
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    (data,target) : tuple if return_X_y == true
```

Comment:
hmm not sure, I'm not super familiar with the intricacies of sphinx and the doc building process. what are your versions (run the snippet below) + sphinx version?
```
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import sklearn; print("Scikit-Learn", sklearn.__version__)
```